tests/nightly/apache_rat_license_check/.rat-excludes

```diff
@@ -0,0 +1,54 @@
+.*xml
+\..*
+.*css
+\\.*
+.*ipynb
+.*html
+.*json
+.*js
+.*txt
+.*md
+3rdparty/*
+R-package/*
+src/operator/mkl/*
+trunk/*
+docker/*
+docker_multiarch/*
+.*\\.m
+.*\\.mk
+.*\\.R
+contrib/*
+Dockerfile*
+.*svg
+.*cfg
+.*config
+docs/*
+__init__.py
+build/*
+.*\\.t
+perl-package/*
```

Comment:
Ignoring the entire perl-package?

Reply:
Yea, these are packages I am not sure about the licensing, I have added details here - 
https://cwiki.apache.org/confluence/display/MXNET/MXNet+Source+Licenses

Reply:
@sergeykolychev any comment about adding apache license to these files?

Reply:
I'll take care of these